src/kratos.py

```diff
@@ -137,6 +138,25 @@ def run_migration(self, timeout: float = 120) -> str:
         ]
         return self._run_cmd(cmd, timeout=timeout)
 
+    def get_version(self) -> Dict[str, str]:
+        """Get the version of the hydra binary."""
+        cmd = ["kratos", "version"]
+
+        stdout = self._run_cmd(cmd)
+
+        # Output has the format:
+        # Version:    {version}
+        # Build Commit:   {hash}
+        # Build Timestamp: {time}
+        out_re = r"Version:[ ]*(.+)\nBuild Commit:[ ]*(.+)\nBuild Timestamp:[ ]*(.+)"
+        versions = re.findall(out_re, stdout)[0]
+
+        return {
```

Comment:
Nitpick:

I think we can just return what the caller needs: the `version`. The callers don't care about other information and they don't necessarily need to know how to obtain it (e.g. do a dict lookup).

Reply:
Fair point